.github/workflows/choose_rc_commit.yml

```diff
@@ -0,0 +1,96 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# This workflow will choose a commit to be the basis of a release candidate
+## and push a new tagged commit for that RC.
+
+# To learn more about GitHub Actions in Apache Beam check the CI.md
+
+name: Choose RC Commit
+on:
+  workflow_dispatch:
+    inputs:
+      RELEASE:
+        description: Beam version of current release
+        required: true
+      RC:
+        description: RC version for the release
+        required: true
+      COMMIT:
+        description: Selected commit to create the release
+        required: true
+      PUSH_TAG:
+        description: Push tag (yes/no)
+        required: false
+        default: no
+      OVERWRITE:
+        description: Overwrite RC Tag (yes/no)
+        required: false
+        default: no
+      DEBUG:
+        description: Enable debug flag adding --debug
+        required: false
+        default: ""
```

Comment:
It was unclear from the description that "--debug" and "" are the only valid inputs. Could we change this to a (yes/no) option like the other inputs?